server/src/main/java/edu/sjsu/moth/server/service/StatusService.java

```diff
@@ -44,13 +44,16 @@ public Mono<Status> findStatusById(String id) {
     }
 
     public Mono<List<Status>> getTimeline(Principal user, String max_id, String since_id, String min_id, int limit) {
-        // TODO: this is an intial hacked implementation. dumps all the statuses
         var qStatus = new QStatus("start");
         var predicate = qStatus.content.isNotNull();
         predicate = addRangeQueries(predicate, max_id, since_id, max_id);
-        return statusRepository.findAll(predicate, Sort.by(Sort.Direction.DESC, "id")).take(limit).collectList();
+        var external = externalStatusRepository.findAll(predicate, Sort.by(Sort.Direction.DESC, "id")).take(limit);
+        var internal = statusRepository.findAll(predicate, Sort.by(Sort.Direction.DESC, "id")).take(limit);
+
+        return Flux.merge(external, internal).collectList();
```

Comment:
you might leave a // TODO here that we may want to merge sort them. or will Flux.merge do that?